drivers/google/option_test.go

```diff
@@ -51,4 +54,7 @@ func TestOptions(t *testing.T) {
 	if got, want := p.zone, "us-central1-f"; got != want {
 		t.Errorf("Want zone %q, got %q", want, got)
 	}
+	if got, want := p.scopes, strings.Split("scope1,scope2", ","); ! cmp.Equal(got, want) {
```

Comment:
I recommend changing scopes to a `[]string` in the `Config` struct.  The library we use supports parsing comma-separated environment variables into an array automatically.
https://github.com/drone/autoscaler/blob/master/config/config.go#L154